views/templates/AdminAccessController/index.tpl

```diff
@@ -1,16 +1,12 @@
 <?php use oat\tao\helpers\Template;?>
 <div class="permission-container flex-container-full">
-    <h1><?= __('Access Permissions for') ?> <em><?= get_data('label') ?></em></h1>
+    <h2><?= __('Access Permissions for') ?> <em><?= get_data('label') ?></em></h2>
```

Comment:
What is the reason behind this change?

Reply:
Getting space and using the same style than everywhere else by the way...